src/daft-sql/src/modules/temporal.rs

```diff
@@ -37,6 +38,7 @@ impl SQLModule for SQLModuleTemporal {
         parent.add_fn("nanosecond", SQLNanosecond);
         parent.add_fn("year", SQLYear);
         parent.add_fn("time", SQLTime);
+        parent.add_fn("to_unix_epoch", SQLUnixTimestamp);
 
         // TODO: Add truncate
         // Our `dt_truncate` function has vastly different semantics than SQL `DATE_TRUNCATE` function.
```

Comment:
This is interesting, and so I'm curious what testing in both directions looks like for timestamp too. I'll have to check the tests, but was thinking if we have a TIMESTAMP with nanoseconds precision, the do `to_unix_epoch(ts, 'seconds') vs the other direction, how is the padding and truncation handled, just zeroes and drop? I'm not sure how else it would possibly be, just want to confirm.